dvc/stage.py

```diff
@@ -222,8 +225,11 @@ def _changed_outs(self):
         for out in self.outs:
             if out.changed():
```

Comment:
Same here.

Reply:
@efiop, I have changed the code to only use status() ones. Are there anything else that needs to be changed? :)